DDG4/python/DDSim/DD4hepSimulation.py

```diff
@@ -108,9 +108,6 @@ def __init__(self):
 
     self._argv = None
 
-    # use TCSH geant UI instead of QT
-    os.environ['G4UI_USE_TCSH'] = "1"
```

Comment:
Is this no longer used by Geant4?
If it is still used, please keep it if `runType != 'qt'`
This prevented many a crash in the past.